react/ModalTrigger.tsx

```diff
@@ -20,20 +31,31 @@ const ModalTrigger: React.FC<{}> = ({ children }) => {
     }
   }
 
-  return (
-    // eslint-disable-next-line jsx-a11y/click-events-have-key-events
-    <div
-      tabIndex={0}
-      role="button"
-      onClick={handleModalOpen}
-      className={`${handles.triggerContainer} bg-transparent pa0 outline-0 bw0`}
-    >
-      {children}
-    </div>
-  )
+  useEffect(() => {
+    if (!openOnLoad && trigger === TriggerMode.load && dispatch) {
+      dispatch({ type: 'OPEN_MODAL' })
+      setOpenOnLoad(true)
+    }
+  }, [trigger, dispatch, openOnLoad])
+
+  if (trigger === TriggerMode.click) {
+    return (
+      // eslint-disable-next-line jsx-a11y/click-events-have-key-events
```

Comment:
Why not use key events in this case?

Reply:
I can't tell which key this would be to open a modal and we might have multiple modals in the page, so this can get complicated

Reply:
The keyboards events of onKeyDown is based on element focus. Will not have ambiguity.